From: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Date: Mon, 16 Mar 2026 10:31:00 +0800
Subject: man-arithmetic

# DP: document deprecated syntax for arithmetic evaluation.

# DP: document deprecated syntax for arithmetic evaluation.
---
 doc/bash.1 | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/doc/bash.1 b/doc/bash.1
index 5af3492..8177174 100644
--- a/doc/bash.1
+++ b/doc/bash.1
@@ -4190,6 +4190,9 @@ The format for arithmetic expansion is:
 \fB$((\fP\fIexpression\fP\fB))\fP
 .RE
 .PP
+The old format \fB$[\fP\fIexpression\fP\fB]\fP is deprecated and will
+be removed in upcoming versions of bash.
+.PP
 The
 .I expression
 undergoes the same expansions
